ts/src/hitbtc.ts

```diff
@@ -2131,19 +2145,15 @@ export default class hitbtc extends Exchange {
         } else if ((type === 'stopLimit') || (type === 'stopMarket') || (type === 'takeProfitLimit') || (type === 'takeProfitMarket')) {
             throw new ExchangeError (this.id + ' createOrder() requires a stopPrice parameter for stop-loss and take-profit orders');
         }
-        let marketType = undefined;
-        [ marketType, params ] = this.handleMarketTypeAndParams ('createOrder', market, params);
-        let method = this.getSupportedMapping (marketType, {
-            'spot': 'privatePostSpotOrder',
-            'swap': 'privatePostFuturesOrder',
-            'margin': 'privatePostMarginOrder',
-        });
-        const [ marginMode, query ] = this.handleMarginModeAndParams ('createOrder', params);
-        if (marginMode !== undefined) {
-            method = 'privatePostMarginOrder';
+        params = this.omit (params, [ 'triggerPrice', 'timeInForce', 'stopPrice', 'stop_price', 'reduceOnly', 'postOnly' ]);
+        let response = undefined;
+        if ((marketType === 'margin') || (marginMode !== undefined)) {
```

Comment:
futures also accept a marginMode parameter so we gotta be careful with these statements, meaning that if marginMode !== undefined does not mean that's automatically spot-margin

Reply:
ok, thanks, in the future, can you just leave a comment and let me make the change myself? If I make the actual code change myself, rather than just reading it, it will help with remembering to check for it again in the future